ill-fated-ducks/Bangazon-Site
# Mb sell

## Status
Choose:
**READY**

## Description
An authenticated user can now add a product to sell and view its details upon submission. 
Inputs are validated in real time instead of on submit. This has been okayed by the Product Owner.

## Todos

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
1. git fetch
2. git checkout ...
3. login as new or existing user
4. Click Sell new product in the navbar
5. Enter product information, try putting strings into int's etc. to test validation
6. Submit new product and ensure it goes to that Product's details and shows up correctly in the database.

## Impacted Areas in Application
List general components of the application that this PR will affect:
See above. Also removed some extra commented code on the default .net homepage

## Link to Feature Ticket
#1
@ill-fated-ducks
